### PR TITLE
Fix tcp socket issue

### PR DESCRIPTION
**Pull request by cliping**
Signed-off-by: Liping Cheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:
#avocado run --vt-type libvirt --vt-machine-type q35 virsh.remote_with_unix.negative_testing.no_allowed_sasl_user --vt-connect-uri qemu:///system
JOB ID     : 32223efc2a3f506f94e2143af7aebfe5e8669c67
JOB LOG    : /root/avocado/job-results/job-2020-06-20T05.23-32223ef/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.remote_with_unix.negative_testing.no_allowed_sasl_user: PASS (169.04 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 170.66 s

File: virttest/utils_conn.py [hunk 1]
```diff
@@ -743,7 +743,11 @@ def conn_recover(self):
                                             r"[\#\$]\s*$")
             libvirtd_service = utils_libvirtd.Libvirtd(session=session)
             if libvirt_version.version_compare(5, 6, 0, session):
+                process.run("systemctl stop libvirtd.socket")
+                libvirtd_service.stop()
                 session.cmd("systemctl stop libvirtd-tcp.socket")
+                process.run("systemctl daemon-reload")
```
Comment (Yingshun) [inline virttest/utils_conn.py:749]: @cliping  it looks like you should execute new added comments in session. So 'process.run' should be 'session.cmd', right?

File: virttest/utils_conn.py [hunk 2]
```diff
@@ -828,7 +832,9 @@ def conn_setup(self):
                 # Before start libvirtd-tcp.socket, user must stop libvirtd.
                 # After libvirtd-tcp.socket is started, user mustn't start
                 # libvirtd.
+                process.run("systemctl stop libvirtd.socket")
                 libvirtd_service.stop()
+                process.run("systemctl daemon-reload")
```
Comment (Yingshun) [inline virttest/utils_conn.py:837]: Some as above.

File: virttest/utils_conn.py (hunk 1 above)
Comment (cliping) [reply virttest/utils_conn.py]: Updated. Thanks.

Comment (Yingshun) [reply virttest/utils_conn.py]: I remember that your original fix was stopping libvirtd.socket and reloading deamon in both conn_recover  and conn_setup. Is the updates in conn_recover() removed by mistake?

File: virttest/utils_conn.py [hunk 3]
```diff
@@ -743,7 +743,10 @@ def conn_recover(self):
                                             r"[\#\$]\s*$")
             libvirtd_service = utils_libvirtd.Libvirtd(session=session)
             if libvirt_version.version_compare(5, 6, 0, session):
+                session.cmd("systemctl stop libvirtd.socket")
+                libvirtd_service.stop()
                 session.cmd("systemctl stop libvirtd-tcp.socket")
+                session.cmd("systemctl daemon-reload")
```
Comment (Yingshun) [inline virttest/utils_conn.py:749]: @cliping  is it necessary to start libvirtd.socket after deamon-reload?

**Pull request closed without merging**